### DISCUSSION TITLE
safe to always run kong migrations bootstrap?

### DISCUSSION THREAD

**Discussion by nwsparks** [Kong/kong#9391] [Help]
Hi, I'm wondering if it is safe to always start kong with `kong migrations bootstrap` even after the database has already been initialized?

My use case is that I am trying to avoid need to change the init containers launch command to `kong migrations up` after an initial launch.

**Comment by StarlightIbuki** [comment]
The command returns `0` when `"Database already bootstrapped"`. But there's no promise about the behavior.
